### PR TITLE
fix: convert yql jsondocument

### PR DESCRIPTION
**Pull request by itwillwork**

### CODE REVIEW

File: lib/converters/yql-to-unipika.js [hunk 1]
```diff
@@ -18,6 +18,11 @@
         var type = wrapYQLType(dataType.toLowerCase());
 
         switch (dataType) {
+            case 'JsonDocument':
+                return {
+                    $type: 'yql.json',
```
Comment (Raubzeug) [inline lib/converters/yql-to-unipika.js:23]: What if we do it [here](https://github.com/gravity-ui/unipika/blob/90d24ab6390f0717dc836f7528f272b854fe7cd2/lib/converters/yql-to-unipika.js#L407)? 
Like
```
case 'JsonDocument':
    return convertSimpleType(data, 'json');
```

File: test/converters/yql-to-unipika.test.ts [hunk 2]
```diff
@@ -62,6 +62,23 @@ describe("converters", function () {
       });
     });
 
+    test("JsonDocument", function () {
```
Comment (Raubzeug) [inline test/converters/yql-to-unipika.test.ts:65]: Please also add [example](https://github.com/gravity-ui/unipika/tree/90d24ab6390f0717dc836f7528f272b854fe7cd2/example).

**Review by Raubzeug — CHANGES_REQUESTED** [review]

File: test/converters/yql-to-unipika.test.ts (hunk 2 above)
Comment (itwillwork) [reply test/converters/yql-to-unipika.test.ts:65]: added
https://github.com/gravity-ui/unipika/pull/32/commits/e3539d939871ce7f894c1fdc748b35ccbc00da60

File: lib/converters/yql-to-unipika.js (hunk 1 above)
Comment (itwillwork) [reply lib/converters/yql-to-unipika.js]: I agree
https://github.com/gravity-ui/unipika/pull/32/commits/fa818d2b6b5024d4615493ce5827d78f4bcd63a3

**Review by Raubzeug — APPROVED** [review]

**Pull request merged**